packages/dev-approuter/README.md

```diff
@@ -0,0 +1,179 @@
+# dev-approuter
+
+> :wave: This is a **community project** and there is no official support for this package! Feel free to use it, open issues, contribute, and help answering questions.
+
+The `dev-approuter` is a dev time wrapper for the [SAP Approuter](https://www.npmjs.com/package/@sap/approuter) that can serve [UI5](https://ui5.sap.com/) and [SAP CAP](https://cap.cloud.sap/docs/) apps that are added as (dev)dependencies to the approuter's `package.json`. A few key notes to begin with:
+- The `dev-approuter` utilizes the [SAP Approuter's extension API](https://help.sap.com/docs/btp/sap-business-technology-platform/extension-api-of-application-router) by adding UI5 servers as extensions - providing the full [UI5 Tooling](https://sap.github.io/ui5-tooling/v3/) experience.
+- A linked SAP CAP app is started on a different port - this is to mimic a deployed architecture. The corresponding destination is automatically created for you.
+- In order to safely separate development configuration from productive code, the `dev-approuter` introduces the concept of the `xs-dev.json` - think of it as an extension to the [`xs-app.json`](https://help.sap.com/docs/btp/sap-business-technology-platform/routing-configuration-file).
+- As the name suggests, the `dev-approuter` is for development only and not meant to be used in production.
+
+---
+
+1. [Prerequisites](#prerequisites)
+2. [Starting the `dev-approuter`](#starting-the-dev-approuter)
+3. [Adding and serving apps](#adding-and-serving-apps)
+    - [UI5 apps](#ui5-apps)
+    - [SAP CAP apps](#sap-cap-apps)
+4. [The `xs-dev.json` file](#the-xs-devjson-file)
+5. [Using the `dev-approuter` and SAP Approuter simultaneously](#using-the-dev-approuter-and-sap-approuter-simultaneously)
+6. [Extending the `dev-approuter`](#extending-the-dev-approuter)
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/en) version 18 or higher.
+
+## Starting the `dev-approuter`
+
+The `dev-approuter` is a wrapper for the SAP Approuter, meaning your current (productive) approuter configuration will also work with the `dev-approuter`, with the option to add dev time configuration to it (see [xs-dev.json](#the-xs-devjson-file)).
+
+1. Install the `dev-approuter` as a dev dependency: 
+    ```bash
+    npm install dev-approuter --save-dev
+    ```
+
+2. Add the following script to the `scripts` section of your approuter's `package.json` file:
+    ```json
+    {
+        ...
+        "scripts": {
+            ...
+            "dev": "node node_modules/dev-approuter"
```

Comment:
maybe in future we just add a binary for this? 

"dev": "dev-approuter"

Reply:
Hm, thought about that, too, but the approuter also just does `node ...`

Reply:
I know - that's why I said -> future! Make the dev-approuter nicer... ;-)